resources/views/masjid_profile/edit.blade.php

```diff
@@ -21,4 +31,125 @@
         </div>
     </div>
 </div>
+<div class="modal fade" id="modal-masjid" tabindex="-1" role="dialog" aria-labelledby="modalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="modalLabel">{{__('masjid_profile.edit')}}</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true"></span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <div class="img-container">
+              <div class="row justify-content-center text-center">
+                  <div class="col-md-8 justify-content-center text-center">
+                      <img id="image" src="" alt="{{ Setting::get('masjid_name') ?? 'buku masjid'}}">
+                  </div>
+                  <div class="col-md-4 justify-content-center text-center">
+                      <div class="preview"></div>
+                  </div>
+              </div>
+          </div>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">{{__('app.cancel')}}</button>
+          <button type="button" class="btn btn-primary" id="crop">{{__('app.crop')}}</button>
+        </div>
+      </div>
+    </div>
+</div>
 @endsection
+
+@section('styles')
+    {{ Html::style(url('css/plugins/cropper.css')) }}
+@endsection
+
+@push('scripts')
+    {{ Html::script(url('js/plugins/cropper.js')) }}
+    {{ Html::script(url('js/plugins/noty.js')) }}
```

Comment:
Kita pakai yang dari CDN saja mas, karena ini hanya di pakai di 1 tempat.

```suggestion
@section('styles')
    {{ Html::style(url('https://cdnjs.cloudflare.com/ajax/libs/cropperjs/1.5.6/cropper.min.css')) }}
@endsection

@push('scripts')
    {{ Html::script(url('https://cdnjs.cloudflare.com/ajax/libs/cropperjs/1.5.6/cropper.min.js')) }}
```